deployment_scripts/deploy_local_with_migration.ts

```diff
@@ -2,9 +2,9 @@ import {Coins, Key, LocalTerra, MnemonicKey, MsgSend} from "@terra-money/terra.j
 import {executeContract, instantiateContract, migrateContract, storeCode} from "./common";
 const path = require('path');
 
-const TERRA_ORIGINAL_HUB_CONTRACT = "lido_terra_hub.wasm"
-const TERRA_ORIGINAL_REWARD_CONTRACT = "lido_terra_reward.wasm"
-const TERRA_ORIGINAL_TOKEN_CONTRACT = "lido_terra_token.wasm"
+const TERRA_ORIGINAL_HUB_CONTRACT = "anchor_basset_hub.wasm"
+const TERRA_ORIGINAL_REWARD_CONTRACT = "anchor_basset_reward.wasm"
+const TERRA_ORIGINAL_TOKEN_CONTRACT = "anchor_basset_token.wasm"
```

Comment:
old contracts name

Reply:
Nope, these are names of the original Terra contracts. This script deploys original contracts at first and then migrates them to the Lido contracts (you can see right Lido names on lines 62-67)